lte/cloud/go/services/subscriberdb/subscriberdb/main.go

```diff
@@ -54,11 +54,15 @@ func main() {
 		glog.Fatalf("Error initializing IP lookup storage: %v", err)
 	}
 
+	// Load service configs
```

Comment:
Nit: I know Orc8r codebase does a bad job of this, but "topic sentence" comments are a bad code smell https://magma.github.io/magma/docs/next/contributing/contribute_conventions#comment-with-why-not-what